src/main/java/echoserver/EchoServerSetup.java

```diff
@@ -0,0 +1,17 @@
+package echoserver;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class EchoServerSetup {
+    public void run(final Integer PORT) throws IOException {
+        ServerSocket hostSocket = new ServerSocket(PORT);
+        HostServer hostServer = new HostServer(hostSocket);
+        Presenter hostScreen = new Presenter(new Console(System.out));
+        EchoServer echoServer = new EchoServer(hostServer, hostScreen);
+
+        hostScreen.showAwaitingInputMessageOnPort(PORT);
+        echoServer.run();
+        hostServer.close();
```

Comment:
You might look at having `HostServer` extend `AutoClosable` so that you can use try-with-resources.